app/models/wpcc/full_contribution_calculator.rb

```diff
@@ -5,12 +5,17 @@ def eligible_salary
     end
 
     def employee_percent
-      1
+      below_threshold ? 3 : 1
     end
 
     def employer_percent
-      return 0 if salary_per_year < lower_earnings_threshold
-      return 1 if salary_per_year >= lower_earnings_threshold
+      below_threshold ? 0 : 1
```

Comment:
Is this likely to change again? Is it worth setting up a config file for this `constant`?

Reply:
I don't know how often it's likely to change. I'm happy to create the constant for it.